docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,153 @@
+---
+id: dual-region
+title: "Dual-Region"
+sidebar_label: "Dual-Region"
+description: "Dual-Region allows you to run Camunda Platform connected in two regions."
+---
+
+import DualRegion from "./img/dual-region.svg";
+
+## Introduction
+
+Camunda 8 is compatible with a dual-region setup under certain [limitations](#limitations). This allows Camunda 8 to run in a mix of active-active and active-passive setups, resulting in an overall **active-passive** setup. The following will explore the concept, limitations, and considerations.
+
+You should get yourself familiar with the topic, the [limitations](#limitations) of Camunda 8 but also general [considerations](#considerations) on operating a dual-region setup.
+
+## Disclaimer
+
+:::danger
+
+- Customers must develop and test operational procedures in non-production environments based on the framework steps outlined by Camunda.
+- Before advancing to production go-live, customers need to validate these procedures with Camunda.
+
+:::
+
+## Architecture
+
+<DualRegion />
+
+The current architecture consists of the following supported components:
```

Comment:
Let's add here that 2 regions mean 2 separate Kubernetes clusters able to communicate to each other (from the image, it's not immediately obvious)